lighthouse-cli/test/fixtures/seo/seo-tap-targets.html

```diff
@@ -108,13 +111,19 @@ <h1>SEO Tap targets</h1>
     </div>
 
     <!-- Links in text blocks are exempted from size/overlap requirements and should not fail -->
-    <p style="width: 100px">
+    <p style="width: 100px;margin-bottom: 30px;">
       This is <a>a link in a text block.</a>
       This is <a>a link in a text block.</a>
       This is <a>a link in a text block.</a>
       This is <a>a link in a text block.</a>
       This is <a>a link in a text block.</a>
     </p>
+
+    <a style="background: #afa; top: 0; left: 0; display: block; width: 100vw; height: 800px;">
```

Comment:
what are `top` and `left` doing here? isn't this element `position: static` so they will have no effect?

Reply:
Good catch, they're not relevant.